packages/shared-ui/src/flowHandler/flows/signup/signupWithPasskeyAppend.ts

```diff
@@ -151,12 +137,7 @@ export const SignupWithPasskeyAppendFlow: Flow = {
       case FlowHandlerEvents.ShowBenefits:
         return FlowUpdate.navigate(ScreenNames.PasskeyBenefits);
       case FlowHandlerEvents.PrimaryButton: {
-        const res = await appendPasskey(state.corbadoApp);
-        if (res.ok) {
-          return res.val;
-        }
-
-        return;
+        return await appendPasskey(state.corbadoApp, state.flowOptions.retryPasskeyOnError);
```

Comment:
While the other two screens work, for PasskeyError screen we have the following problem: When you start another retry there the loading button will not stop loading (reason for that is that there is no state update here when the passkey operation is stopped).
![image](https://github.com/corbado/javascript/assets/134682658/eed85c32-e25e-4be4-a914-5c3c1030a22a)